.gitignore

```diff
@@ -36,4 +36,5 @@ bitwarden.env
 next-env.d.ts
 
 # linting / formatting
-.eslintcache
\ No newline at end of file
+.eslintcache
+settings.json
```

Comment:
Hmmm... it seems like `settings.json` was still pushed, maybe it was staged for commit before the `.gitignore` was updated?